### DISCUSSION TITLE
Forcing a full scan

### DISCUSSION THREAD

**Discussion by addohm** [abraunegg/onedrive#2631] [Q&A] [answered]
Is there a way to force a full scan without a full sync?  I want to make sure the files on the physical machine are fully matched up with the remote after having some local machine issues.

**Comment by abraunegg** [comment, marked as answer]
As advised before (https://github.com/abraunegg/onedrive/discussions/2600) the option to do a full scan is a `--resync`

My suggestion here is that you also look to build an 'alpha-5' Docker container for performance reasons as per previously suggested.

**Reply by addohm** [reply]
Sorry, I'm brain dead.  I really don't remember havign asked this already 🍡
